su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,16 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <h2>
+      <a name="<%= name.gsub(' ', '_').downcase %>">
```

Comment:
I like it. I've also seen the pilcrow **¶** and section **§** symbols used for simialar functionality.